aspnetcore/fundamentals/environments/3.1sample/EnvironmentsSample/Startup.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace First
+{
+    #region snippet_all
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
```

Comment:
It doesn't look like any of this gets used, so removing it would make for a smaller snippet. It looks like this is true for the other `Startup` classes, too.